resources/travis/linux_install.py

```diff
@@ -0,0 +1,71 @@
+import os
+import io
+import sys
+import zipfile
+
+from urllib import request
+from pathlib import Path
+from zipfile import ZipFile, ZipInfo
+from urllib.error import URLError
+
+HOME_PATH: Path = Path.home()
+
+
+class ZipFileWithPermissions(ZipFile):
+    """Custom ZipFile class handling file permissions.
+
+    Code from https://stackoverflow.com/a/54748564
+    """
+
+    def _extract_member(self, member, targetpath, pwd):
+
```

Comment:
Can you double check the format of this file?

Reply:
I'm not sure what you mean...